search.html

```diff
@@ -0,0 +1,263 @@
+							<li><a href="apps/ftp/index.html">ftp</a></li>
+							<li><a href="apps/inets/index.html">inets</a></li>
+							<li><a href="apps/jinterface/index.html">jinterface</a></li>
+							<li><a href="apps/megaco/index.html">megaco</a></li>
+							<li><a href="apps/public_key/index.html">public_key</a></li>
+							<li><a href="apps/ssh/index.html">ssh</a></li>
+							<li><a href="apps/ssl/index.html">ssl</a></li>
+							<li><a href="apps/tftp/index.html">tftp</a></li>
+							<li><a href="apps/wx/index.html">wx</a></li>
+							<li><a href="apps/xmerl/index.html">xmerl</a></li>
+						</ul>
+					</li>
+					<li>Tools
+						<ul>
+							<li><a href="apps/debugger/index.html">debugger</a></li>
+							<li><a href="apps/dialyzer/index.html">dialyzer</a></li>
+							<li><a href="apps/et/index.html">et</a></li>
+							<li><a href="apps/observer/index.html">observer</a></li>
+							<li><a href="apps/parsetools/index.html">parsetools</a></li>
+							<li><a href="apps/reltool/index.html">reltool</a></li>
+							<li><a href="apps/runtime_tools/index.html">runtime_tools</a></li>
+							<li><a href="apps/syntax_tools/index.html">syntax_tools</a></li>
+							<li><a href="apps/tools/index.html">tools</a></li>
+						</ul>
+					</li>
+					<li>Test
+						<ul>
+							<li><a href="apps/common_test/index.html">common_test</a></li>
+							<li><a href="apps/eunit/index.html">eunit</a></li>
+						</ul>
+					</li>
+					<li>Documentation
+						<ul>
+							<li><a href="apps/edoc/index.html">edoc</a></li>
+							<li><a href="apps/erl_docgen/index.html">erl_docgen</a></li>
+						</ul>
+					</li>
+				</ul>
+			</nav>
+		</aside>
+		<div id="content">
+			<div class="innertube">
+				<p class="main-description">Erlang/OTP documentation search</p>
+				<div class="search-box" id="searchbox"></div>
+				<div class="search-hits" id="hits"></div>
+                 <div class="search-pagination" id="pagination"></div>
+				<center> 
+                    <small>
+                          Copyright &copy; 1999-2021
+                          <a href="http://www.ericsson.com">Ericsson AB</a>
+                      </small> 
+                </center>
+			</div>
+		</div>
+	</div>
+
+	<script src="/assets/js/doc-search.bundle.js"></script>
+
+	<script type=text/html id=hit-template>
+		<a href="{{ url }}" class=hit>
+			<div class=hit-content>
+				<h3 class="hit-name lvl0">{{{_highlightResult.hierarchy.lvl0.value}}}</h3>
+				<h4 class="hit-description lvl1">{{{_highlightResult.hierarchy.lvl1.value}}}</h4>
+				<h5 class="hit-description lvl2">{{{_highlightResult.hierarchy.lvl2.value}}}</h5>
+				<div class=hit-text>{{{_highlightResult.content.value}}}</div>
+				<div class=hit-tag>{{ tags }}</div>
+			</div>
+		</a>
+	</script>
+	
+    <script>
+        const searchClient = algoliasearch('T71IQF5OFV', 'a2c3c6c0120e6eb2574bb328bd316967');
+	    const search = instantsearch({
+		    indexName: 'erlang_org',
```

Comment:
```suggestion
        const searchClient = algoliasearch('LUYTU1J2MB', '86152ba1d4a9d7e179d537b8060a4c31');
	    const search = instantsearch({
		    indexName: 'erlang',
```

Use these keys instead, the other ones are old keys.